src/ECER.Clients.RegistryPortal/ECER.Clients.RegistryPortal.Server/Applications/ApplicationsEndpoints.cs

```diff
@@ -124,6 +126,10 @@ public record Transcript()
   [Required]
   public DateTime EndDate { get; set; }
 }
+public record WorkExperienceReference(string? Id, [Required] string? FirstName, [Required] string? LastName, [Required] string? EmailAddress)
```

Comment:
@farzadnadiri should we have id in body?

Reply:
Yes, we should.